inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/iceberg/IcebergSinkOperation.java

```diff
@@ -133,6 +136,26 @@ public void saveFieldOpt(SinkRequest request) {
         LOGGER.info("success to save iceberg field");
     }
 
+    private void checkFieldInfo(SinkField field) {
+        if (FieldType.forName(field.getFieldType()) == FieldType.DECIMAL) {
+            IcebergColumnInfo info = IcebergColumnInfo.getFromJson(field.getExtParams());
+            if (info.getPrecision() == null || info.getScale() == null) {
+                String errorMsg = String.format("precision or scale not specified for decimal field (%s)",
+                        field.getFieldName());
+                LOGGER.error("field info check error: {}", errorMsg);
+                throw new IllegalArgumentException(errorMsg);
```

Comment:
Please change the exception to `BusinessException` which it can be recognized by the front end.